src/components/Reports/GoalCalculator/SharedComponents/GoalCalculatorGrid/GoalCalculatorGrid.test.tsx

```diff
@@ -179,19 +179,82 @@ describe('GoalCalculatorGrid', () => {
     expect(await findByText('$1,800.00')).toBeInTheDocument();
   });
 
-  it('toggles direct input switch', async () => {
-    const { getByRole } = render(
+  it('toggles between Lump Sum and Line Item buttons', async () => {
+    const {
+      getByText,
+      findByLabelText,
+      queryByRole,
+      findByRole,
+      queryByLabelText,
+      getByRole,
+    } = render(
       <TestWrapper>
         <GoalCalculatorGrid {...defaultProps} />
       </TestWrapper>,
     );
 
-    const directInputSwitch = getByRole('checkbox');
-    expect(directInputSwitch).not.toBeChecked();
-    userEvent.click(directInputSwitch);
-    expect(directInputSwitch).toBeChecked();
-    userEvent.click(directInputSwitch);
-    expect(directInputSwitch).not.toBeChecked();
+    const lumpSumButton = getByText('Lump Sum');
+    const lineItemButton = getByText('Line Item');
+    expect(queryByLabelText('Total')).not.toBeInTheDocument();
+    expect(getByRole('grid')).toBeInTheDocument();
+    userEvent.click(lumpSumButton);
+    expect(await findByLabelText('Total')).toBeInTheDocument();
+    await waitFor(() => {
+      expect(queryByRole('grid')).not.toBeInTheDocument();
+    });
```

Comment:
It doesn't look like `waitFor` is needed here.

```suggestion
    expect(queryByRole('grid')).not.toBeInTheDocument();
```